src/app/account-settings/components/settings-actions-duplicated/settings-actions-duplicated.component.ts

```diff
@@ -93,8 +93,13 @@ export class SettingsActionsDuplicatedComponent implements OnInit {
             .afterClosed()
         })
       )
-      .subscribe(() => {
-        this.loading.next(false)
-      })
+      .subscribe(
```

Comment:
do we need duplicated the `loading.next`

Reply:
Yes! 
When the observer `complete` or `error` we need the loading to stop

Reply:
Great catch!